hoot-core/src/main/cpp/hoot/core/conflate/network/OsmNetworkExtractor.cpp

```diff
@@ -101,7 +101,7 @@ bool OsmNetworkExtractor::_isContiguous(const ConstRelationPtr& r)
 
   const vector<RelationData::Entry>& members = r->getMembers();
   bool result = members.size() > 0;
-  long lastNode;
+  long lastNode = 0;
```

Comment:
This one is actually not possible but it still is good practice to initialize your variables.  Using 0 because it isn't a valid ID.